### PR TITLE
[zh-cn] sync /docs/tasks/configure-pod-container/create-hostprocess-p…

### PR DESCRIPTION
**Pull request by huan555**
syn with en.
content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
You can find much more changes by running this command in the repo:
```
./scripts/lsync.sh content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md
```

For localization style of Chinese, refer to https://kubernetes.io/zh-cn/docs/contribute/localization_zh/

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 1]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+HostProcess容器可以从任何现有的[Windows容器基本映像](https://learn.microsoft.com/virtualization/windowscontainers/manage-containers/container-base-images)构建。
+
+此外，还为 HostProcess 容器创建了一个新的基本镜像！
+详细相关信息，请查看[github项目：windows-host-process-containers-base-image](https://github.com/microsoft/windows-host-process-containers-base-image#overview)。
+
+## HostProcess 容器故障排除
+
+- HostProcess 容器无法启动并产生错误 `failed to create user process token: failed to logon user: Access is denied.: unknown`
+
+  确保containerd以 `LocalSystem` 或 `LocalService` 服务帐户运行。用户帐户（甚至管理员帐户）无权为任何受支持的[用户帐户](#choosing-a-user-account)创建登录令牌。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:446]: 參照其他翻譯件，適當折行。

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 2]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+-->
+
+## HostProcess 容器的基本镜像
+
+HostProcess容器可以从任何现有的[Windows容器基本映像](https://learn.microsoft.com/virtualization/windowscontainers/manage-containers/container-base-images)构建。
+
+此外，还为 HostProcess 容器创建了一个新的基本镜像！
+详细相关信息，请查看[github项目：windows-host-process-containers-base-image](https://github.com/microsoft/windows-host-process-containers-base-image#overview)。
+
+## HostProcess 容器故障排除
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:442]: 這個小節可以單獨翻譯，不要批量翻譯太大段落。

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 3]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+For more information please check out the [windows-host-process-containers-base-image github project](https://github.com/microsoft/windows-host-process-containers-base-image#overview).
+
+## Troubleshooting HostProcess containers
+
+- HostProcess containers fail to start with `failed to create user process token: failed to logon user: Access is denied.: unknown`
+
+  Ensure containerd is running as `LocalSystem` or `LocalService` service accounts. User accounts (even Administrator accounts) do not have permissions to create logon tokens for any of the supported [user accounts](#choosing-a-user-account).
+-->
+
+## HostProcess 容器的基本镜像
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:435]: base 在這裡還是“基礎”的意思，譯為“基本”不太合適。

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 4]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+1. Schedule the pod!
+-->
+
+示例:
+
+1. 在节点上创建本地用户组（这可以在另一个 HostProcess 容器中完成）。
+
+    ```cmd
+    net localgroup hpc-localgroup /add
+    ```
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:405]:
````suggestion
   ```cmd
   net localgroup hpc-localgroup /add
   ```
````

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 5]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+1. 向本地用户组授予对节点上所需资源的访问权限。这可以通过 [iccls](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/icacls) 等工具完成。
+
+1. 将 `runAsUserName` 设置为 Pod 或单个容器的本地用户组的名称。
+
+    ```yaml
+    securityContext:
+      windowsOptions:
+        hostProcess: true
+        runAsUserName: hpc-localgroup
+    ```
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:416]:
````suggestion
   ```yaml
   securityContext:
     windowsOptions:
       hostProcess: true
       runAsUserName: hpc-localgroup
   ```
````

File: content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md [hunk 6]
```diff
@@ -344,3 +344,104 @@ use the LocalService service account as it is the least privileged of the three
...
+{{< /note >}}
+
+<!--
+Example:
+
+1. Create a local user group on the node (this can be done in another HostProcess container).
+
+    ```cmd
+    net localgroup hpc-localgroup /add
+    ```
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/create-hostprocess-pod.md:382]: 這裡的代碼段沒有翻譯，所以可以單獨處理。
不要大段大段地翻譯。

**Comment by Sea-n** [comment]
Hey @huan555,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by windsonsea** [comment]
Since 3 weeks passed, the author has no any response, this PR will be closed. You can re-open it when you are available.

/close

**Pull request closed without merging**